pages/project.js

```diff
@@ -67,13 +68,16 @@ export default function ProjectDetails() {
                 ...project,
                 stats,
                 owner,
+                marketplace: marketplaceLinks,
             }
         },
         {
             refetchOnWindowFocus: false,
         }
     )
 
+    if (project) project.status = [{ fully_funded: null }]
```

Comment:
@MaxLascombe not sure what this bit is doing?

Would it not end up setting the project status to `fully_funded` for all projects?

Reply:
Ah, good catch. I was using that for testing purposes. I'll remove it now.